soifunc/resize.py

```diff
@@ -14,21 +14,47 @@
 ]
 
 
+@dataclass
+class GoodScaler(GenericScaler):
+    """High quality resizing filter with opinionated settings"""
+
+    def __post_init__(self) -> None:
+        self.scaler = HybridScaler(SSIM, Spline36)
```

Comment:
You probably want to somehow allow the user to pass on kwargs to at least Spline36 (assuming you want stuff like `keep_ar` to function. 

Consider something like this instead:

```py
def __init__(self, scaler: ScalerT = Spline36) -> None:
    self.scaler = HybridScaler(SSIM, Spline36.from_param(scaler))
```